polaris-react/src/components/KeyboardKey/KeyboardKey.tsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 
+import {classNames} from '../../utilities/css';
+
 import styles from './KeyboardKey.scss';
 
+type Size = 'small';
+
 export interface KeyboardKeyProps {
-  /** The content to display inside the key */
   children?: string;
+  size?: Size;
 }
-
-export function KeyboardKey({children}: KeyboardKeyProps) {
+export function KeyboardKey({children, size}: KeyboardKeyProps) {
   let key = children || '';
-  key = key.length > 1 ? key.toLowerCase() : key.toUpperCase();
+  key = size || key.length > 1 ? key.toLowerCase() : key.toUpperCase();
```

Comment:
And this would become 
```suggestion
  const key = size || children.length > 1 ? children.toLowerCase() : children.toUpperCase();
```

Reply:
Good suggestion! 👌 I also made a small change on the  logic as we actually want the `small` variant to always have upper case